ngraph/python/src/pyngraph/frontend_manager.cpp

```diff
@@ -0,0 +1,320 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl_bind.h>
+
+#include "frontend_manager.hpp"
+#include "frontend_manager/frontend_exceptions.hpp"
+#include "frontend_manager/frontend_manager.hpp"
+#include "pyngraph/function.hpp"
+
+namespace py = pybind11;
+
+void regclass_pyngraph_FrontEndManager(py::module m)
+{
+    py::class_<ngraph::frontend::FrontEndManager,
```

Comment:
Can we add documentation?
Examples you can find here: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/python/src/pyngraph/node.cpp#L75

Reply:
Done, added documentation